### PR TITLE
Cloud organization update

### PR DESCRIPTION
**Pull request by jonat123**
## Description

_What did you add/update/change?_

Based on the following issue: #5396 I have added some more information on what options you have when you are part of an organization on Umbraco cloud.

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
 # Organizations
 
-On Umbraco Cloud, you can get an Umbraco Cloud Organization. With an Organization, you get an overview of all the projects that have been created by you and the members of your Organization.
+On Umbraco Cloud it is possible to get an Organization. An organization is handy if you are a partner or are managing a lot of projects for different customers.
 
-If you are interested in getting an Organization, please reach out to our Support Team through the Chat box in your project overview.
-
-## Managing your Organization
+With an organization you get an overview of all projects and members part of the organization and you manage project payments and other options.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:5]: There are many "and" in this sentence.
```suggestion
With an organization, you get an overview of all projects and members that are part of it, manage the project payments, and many others.
```

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 
 In the following sections, we will go through the different options that are available in an Organization:
 
 ![Organization Overview](images/Cloud-org-overview.png)
 
+{% hint style="info" %}
+If you are interested in getting an Organization, please reach out to our Support Team through the Chat box in your project overview.
+{% endhint %}
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:13]: I found out that if you get the warning that you need to use less than 25 words in an hint note, but you actually do use less than 25 words, then if you add a space before and after the hint the warning will be gone 😁 
```suggestion
{% hint style="info" %}

If you are interested in getting an Organization, please reach out to the Support Team through the Chat box in your [project overview](https://www.s1.umbraco.io/projects).

{% endhint %}
```

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 3]
```diff
@@ -22,7 +25,45 @@ In the **Members** section, you can get an overview, view pending invites, and t
...
 <figure><img src="../../../.gitbook/assets/members.png" alt=""><figcaption></figcaption></figure>
 
-### MFA enforcement
+Members added to your ogranization can see different information about the organization based on the user group they are added to. Currently there are three different groups, **Read**, **Write** and **Admin**.
+Below you can see what each user group has access to under the organization they are a part of.
+
+Organization Members with **Admin Access** can do the following in the organization:
+
+* Update the organization information
+* Invite to the organization
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:34]:
```suggestion
* Invite others to the organization
```

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 4]
```diff
@@ -22,7 +25,45 @@ In the **Members** section, you can get an overview, view pending invites, and t
...
+* Invite Users to project under the organization
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:35]: Hmm, am not sure if you are actually able to invite users to the projects from the organization section. Can you check with Søren to review this PR as well to confirm these details? 

Also, can you ask him as well what is the difference between users and members (as in some cases members are used an others users are used)? I am not sure I understand what is the difference between them in terms of organization 🤔

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 5]
```diff
@@ -47,7 +88,9 @@ In the **Projects** section, you can get an overview of all the Projects that ha
 
 ![Project overview](images/Projects-v10.png)
 
-Additionally, you can view if any Projects have been disabled under your Organization.
+It is also possible to see the plan, project status, payment status, creation date, region and number of environments for each of your projects.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:91]: as there are more options to see in this case, maybe the image above needs to be updated with the new view as well? 💡

**Review by alina-tincas — CHANGES_REQUESTED** [review]
@jonat123 thank you for the PR 😁 

I have made a few suggestions, can you please take a look into them? 💪

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md (hunk 4 above)
Comment (jonat123) [reply umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:35]: Already checked with Søren, and they can indeed invite users to projects :D 

Yeah, I understand the confusion, and I have talked with Søren about this in the past 🙈

Users are invited to cloud projects, and members is when you invite to an organization... In the end it is just a matter of terms, and as long as it is like that in the cloud portal it should also be this way in the docs 😊 I do hope it makes sense :D

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md (hunk 5 above)
Comment (jonat123) [reply umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:91]: That is a good point!

I will update the image!

Comment (jonat123) [reply umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:93]: The image has been updated :-)

Comment (alina-tincas) [reply umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:93]: awesome! Then all is good now 😊

**Review by alina-tincas — APPROVED** [review]

**Comment by jonat123** [comment]
Merging in.

**Pull request merged**